pages/generative-apis/reference-content/adding-ai-to-zed-ide.mdx

```diff
@@ -0,0 +1,60 @@
+---
+meta:
+  title: Adding AI to the Zed IDE using Generative APIs
+  description: Learn how to integrate AI-powered code models into Zed using Scaleway's Generative APIs.
+content:
+  h1: Adding AI to to the Zed IDE using Generative APIs
+  paragraph: Elevate your coding experience by integrating AI-powered code models into the Zed IDE. With Continue and Scaleway's Generative APIs, you can leverage AI to understand, generate, and optimize code with ease.
+tags: generative-apis ai machine-learning language-models code-assistance vs-code continue
+validation_date: 2025-03-18
```

Comment:
I don't think this is the right format for the dates, it's supposed to be

dates:
    validation: xx
    posted: xx